src/main/webapp/app/markdown-editor/commands/orderedListCommand.ts

```diff
@@ -0,0 +1,56 @@
+import { Command } from './command';
+
+export class OrderedListCommand extends Command {
+
+    buttonIcon = 'list-ol';
+    buttonTranslationString = 'arTeMiSApp.multipleChoiceQuestion.editor.orderedList';
+
+    /**
+     * @function execute
+     * @desc Use the markdown language for creating/removing an ordered list
+     */
+    execute(): void {
+        const selectedText = this.getSelectedText();
+        this.splitText(selectedText);
+    }
+
+    /**
+     * @function splitText
+     * @desc 1. Split the text at the line break into an array
+     *       2. Assign each line the position it has in the array
+     *       3. Call for each textline the replaceText method
+     * @param {string} the selected text by the cursor
+     */
+    splitText(selectedText: string): void {
+        const parseArray = selectedText.split('\n');
+        let addAmount = parseArray.length - 1;
+        for (const element of parseArray) {
+             this.replaceText(element, parseArray.length - addAmount);
+             addAmount--;
+        }
+    }
+
+    /**
+     * @function execute
```

Comment:
I think they way you are handling this is very clever!
However I had to read through the code to understand the outcome of the function and am still not sure completely.

What is the meaning of the '.' in the string? Is it just a full stop as in a sentence?
Why does it choose a seemingly arbitrary length of the element of 3?
Why does it first check if element is an empty string and then insert it into the element (line 48)
The jsdoc doesn't clarify what the input of the function means and what the expected outcome is.
E.g. 2. If included reduce the selected text by this element and replace the selected text by textToAdd
This explains the code of the function, but not the expected outcome.

Reply:
Better!